src/transforms/uglify.js

```diff
@@ -1,17 +1,19 @@
-const {AST_Node, minify} = require('uglify-js');
+const {AST_Node, minify} = require('uglify-es');
 const {toEstree} = require('babel-to-estree');
 const types = require('babel-types');
 const walk = require('babylon-walk');
 
 module.exports = async function(asset) {
   await asset.parseIfNeeded();
 
-  // Convert to UglifyJS AST
+  // Convert to UglifyES AST
   var ast = AST_Node.from_mozilla_ast(toEstree(asset.ast, asset.contents));
```

Comment:
This won't work for ES6+ code.

You either have to use the built-in `uglify-es` parser instead of ESTree or add ES6+ support to `uglify-es`.

It's an known issue that only ES5 ESTree constructs are currently converted by `uglify-es`: https://github.com/mishoo/UglifyJS2/issues/968

Reply:
Ow damn that kinda sucks, i'll read through that issue for sure